src/test/java/pl/eukon05/eventboard/event/application/service/UnitTests.java

```diff
@@ -140,4 +144,38 @@ void should_attend_invited_event() {
 
         assertTrue(attendEventUseCase.execute(userID, event.getId()));
     }
+
+    @Test
+    void should_invite_to_public_event() {
+        Event event = createTestEvent();
+        String friendID = "somerandomid";
+        Mockito.when(getEventPort.getEventById(Mockito.anyLong())).thenReturn(Optional.of(event));
+        Mockito.when(getUserFriendlistPort.getUserFriendlist(Mockito.anyString())).thenReturn(List.of(friendID));
+
+        assertTrue(inviteToEventUseCase.execute(userID, friendID, event.getId()));
+        assertTrue(event.getInviteeIDs().contains(friendID));
```

Comment:
Doesn't JUnit have better assertions than `assertTrue`? Maybe you can use `Hamcrest` or write your own assertions?

I would love to see something like:
```suggestion
        assertInviriteeIDsContains(event, friendID);
```